app/uk/gov/hmrc/soletraderidentification/controllers/JourneyDataController.scala

```
@@ -111,7 +115,7 @@ class JourneyDataController @Inject()(cc: ControllerComponents,
             _ => NoContent
```

Should you throw an error if the "removeJourneyData" method returns false?

---

added